Arguments: 
  /usr/local/bin/node /usr/local/Cellar/yarn/1.5.1/libexec/bin/yarn.js start

PATH: 
  /Library/Frameworks/Python.framework/Versions/3.6/bin:/Library/Frameworks/Python.framework/Versions/3.6/bin:/Library/Frameworks/Python.framework/Versions/3.6/bin:/usr/local/bin:/usr/bin:/bin:/usr/sbin:/sbin:/Applications/VMware Fusion.app/Contents/Public

Yarn version: 
  1.5.1

Node version: 
  9.4.0

Platform: 
  darwin x64

npm manifest: 
  {
    "name": "auth-client",
    "version": "0.1.0",
    "private": true,
    "dependencies": {
      "axios": "^0.16.1",
      "react": "^15.5.4",
      "react-dom": "^15.5.4",
      "react-redux": "^5.0.4",
      "react-router-dom": "^4.1.1",
      "redux": "^3.6.0",
      "redux-form": "^6.7.0",
      "redux-thunk": "^2.2.0"
    },
    "devDependencies": {
      "react-scripts": "^1.0.7"
    },
    "scripts": {
      "start": "react-scripts start",
      "build": "react-scripts build",
      "test": "react-scripts test --env=jsdom",
      "eject": "react-scripts eject"
    }
  }

yarn manifest: 
  No manifest

Lockfile: 
  No lockfile

Trace: 
  Error: Command failed.
  Exit code: 127
  Command: sh
  Arguments: -c react-scripts start
  Directory: /Users/SamuelCha/Documents/Lambda School/11/pp/Client-Auth
  Output:
  
      at ProcessTermError.MessageError (/usr/local/Cellar/yarn/1.5.1/libexec/lib/cli.js:186:110)
      at new ProcessTermError (/usr/local/Cellar/yarn/1.5.1/libexec/lib/cli.js:226:113)
      at ChildProcess.<anonymous> (/usr/local/Cellar/yarn/1.5.1/libexec/lib/cli.js:30281:17)
      at ChildProcess.emit (events.js:160:13)
      at maybeClose (internal/child_process.js:943:16)
      at Process.ChildProcess._handle.onexit (internal/child_process.js:220:5)
